go/worker/keymanager/worker.go

```diff
@@ -592,6 +593,32 @@ func (crw *clientRuntimeWatcher) updateExternalServicePolicyLocked(snapshot *com
 
 	crw.w.grpcPolicy.SetAccessPolicy(policy, crw.node.Runtime.ID())
 	crw.w.logger.Debug("worker/keymanager: new normal runtime access policy in effect", "policy", policy)
+
+	// Notify the sentry nodes of the new policy.
+	for _, addr := range crw.w.commonWorker.GetConfig().SentryAddresses {
```

Comment:
This seems like some common functionality, could it be handled in a single place instead of each piece of code creating a new connection to the sentry nodes?

It should work something like this:
* All the different places call `SetAccessPolicy` (as it is now, e.g. see above).
* There is a global `PolicyWatcher` that is used with all such policies. As far as I can see the policy watcher stuff is already done it should just be changed to no longer be exposed over gRPC and only used locally?
* The policy watcher is such that it will notify the sentry nodes over the single connection that it has. This connection should be shared with the registration worker as well as there is no sense in having different connections to the sentry workers from so many places.

Reply:
This is now done and ready for re-review :)

The policy watcher now handles pushing policy updates to the sentry nodes, however, I've left the registry worker as-is, since it only connects to the sentry nodes when rotating certificates and it needs to establish a new connection anyways in this case, since the certs have changed.